R/c.R

```diff
@@ -54,8 +54,9 @@ vec_c <- function(..., .ptype = NULL) {
 
     x <- vec_cast(args[[i]], to = ptype)
```

Comment:
Doesn't this now violate the vctrs invariant that vectors are either all named or all unnamed?

Reply:
The names are restored afterwards: https://github.com/r-lib/vctrs/pull/176/files#diff-8659cf38b76d4f1d2bda41fdb89168d4R64.

Anyway, removing this change for now.